README.md

```diff
@@ -0,0 +1,80 @@
+  * **`options`**: map of additional options to pass to the module.
+
+### alerts
+You can look at the example files in `alerts/` to get a taste of the syntax. You will need the following fields in your alerts file:
+
+* **`name`**: this will be the primary key of your alert, and will show up in emails and in the UI. Only one alert of a given name will be created, so if your alert name is not using some part of the `@hostinfo` object to make it unique, you will end up with a single alert.
+* **`message`**: the body of the alert that will be set to the destinations (currently only Datadog).
+* **`applies`**: determines which hosts the alert applies to; putting `true` will end up creating as many alerts as the number of distinct `name`s your alert expands into. Interferon will iterate through all the hosts generated by `host_sources` and test them agains the expression here.
+* **`notify.people`**: an array of the people who will be receiving the alert.
+* **`notify.groups`**: an array of groups who will be receiving the alert. The group mapping will be based on the output of `group_sources`. The `filesystem` group_source module will read the groups in the `groups` directory.
+* **`metric`**: The Datadog metric alert query.
+* **`silenced`**: The Datadog silenced setting.
+* **`timeout`**: The Datadog timeout setting in seconds (gets converted to `timeout_h`).
+
+### destinations
+This directory contains additional modules for `destinations`.
+
+### groups
+We only have a single group_source called `filesystem`, which reads groups in YAML files from the filesystem (specifically this `groups`). Each file contains the following keys:
+* **`name`**: Name of the group (used in alerts).
+* **`people`**: List of people in the group. 
+* **`alias_for`**: Name of another group that this group is an alias for. Useful for transistion periods when renaming groups.
+
+To post to a slack channel, make sure that the slack integration is configured to include that channel: https://app.datadoghq.com/account/settings#integrations/slack
+
+### group_sources
+This directory contains additional modules for `group_sources`. Each group_source should be a class which implements the following methods.
+* **`intialize(options)`**: Called for initializing the group_source and reading the options passed in from `config.yml`.
+* **`list_groups`**: Called to get the group mapping generated by this module. Should return a hash contain group name to people.
+
+### host_sources
+This directory contains additional modules for `host_sources`. Each host_source should be a class which implements the following methods:
+* **`intialize(options)`**: Called for initializing the group_source and reading the options passed in from `config.yml`.
+* **`list_hosts`**: Called to get the host list generated by this module. Should return a list of hashes that can be referenced using `@hostinfo` in alerts.
+
```

Comment:
maybe add:
```
We've implemented a few AWS-related host sources which use billow.
You can grab billow from https://github.com/airbnb/billow.
We use billow instead of hitting the AWS API directly to avoid hitting rate limits, but you're welcome to implement your own host sources which use whatever APIs you want to learn about the machines available in your infrastructure.
```